### DISCUSSION TITLE
Extension not loading

### DISCUSSION THREAD

**Discussion by cscrum** [bytesandbrains/h3-pg#127] [Q&A] [answered]
I have built the extension using your guide and run pgxn install h3 and that seemed to go fine.

I'm getting this error when trying to load the extension using pgxn load h3 :
INFO: best version: h3 4.1.2
ERROR:  extension "h3" already exists
ERROR:  required extension "postgis" is not installed
HINT:  Use CREATE EXTENSION ... CASCADE to install required extensions too.

I definitely have postgis installed as I've been using it for more than 2 years. I installed postgis from the CLI within postgres, but just for kicks I installed it again using apt. That went fine as well. Any tips here on how to get the h3 loaded?

**Comment by zachasme** [comment]
Hi @cscrum, that does sounds strange!

What version of ubuntu are you running? What is the output of the command `pg_lsclusters`?

How did you install h3? Did you also build it manually using CMake? Or did you only use pgxn?

If you have multiple postgres installations you might have to provide the `--pg_config` argument to pgxnclient, see [docs for `pgxn install`](https://pgxn.github.io/pgxnclient/usage.html#pgxn-install).

**Comment by albjeremias** [comment, marked as answer]
you need to add the extension for each database..
so you open your database.. and run:
`create extension h3`
